README.md

```diff
@@ -11,31 +11,29 @@
 
 ## Staging
 
-- **url**: [https://service-manual.apps.staging.digital.gov.au/](https://service-manual.apps.staging.digital.gov.au/)
-- **branch**: `develop`
-- **status**:  [![CircleCI](https://circleci.com/gh/govau/service-manual/tree/develop.svg?style=svg)](https://circleci.com/gh/govau/service-manual/tree/develop)
```

Comment:
The url and status shouldn't be removed

Reply:
Wait until you have a new application set up in cloud foundry and urls approved etc. before doing these changes.

Reply:
Might just revert it back